src/app/services/ix-slide-in2.service.ts

```diff
@@ -35,8 +37,9 @@ export class IxSlideIn2Service {
 
     const slideInRef = this.slideIn2Component.openSlideIn<T, D>(component, params);
     this.slideInRefMap.set(slideInRef.id, slideInRef);
-    slideInRef.slideInClosed$.pipe(untilDestroyed(this)).subscribe(() => {
+    slideInRef.slideInClosed$.pipe(untilDestroyed(this)).subscribe((response) => {
       this.deleteRef(slideInRef.id);
+      this.onClose$.next(response);
```

Comment:
I don't think we need to pass responses, as it won't be possible to properly filter them.